test/teletype-package.test.js

```diff
@@ -988,6 +988,29 @@ suite('TeletypePackage', function () {
     assert(description.includes('some error'))
   })
 
+  test('resets editor and button after failed sign-in attempt', async () => {
+    const env = buildAtomEnvironment()
+    const pack = await buildPackage(env, {signIn: false})
+    await pack.consumeStatusBar(new FakeStatusBar())
+    pack.client.signIn = async function () {
+      throw new Error('some error')
+    }
+
+    const {popoverComponent} = pack.portalStatusBarIndicator
+    const {signInComponent} = popoverComponent.refs
+
+    signInComponent.refs.editor.setText('some-token')
+
+    await signInComponent.signIn()
+
+    // TODO: Find out why these refs are not updating.
```

Comment:
@jasonrudolph: This test case is supposed to test what happens when a token value is provided, the user clicks sign in, and the sign in fails. Testing using atom, it resets the token field and disables the button. However, it seems that I cannot get the refs here to be updated to actually test how it worked. I have banged my head on this all weekend... Any ideas?